### PR TITLE
[zh] fix typo & link

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/44235/files

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md [hunk 1]
```diff
@@ -77,11 +77,11 @@ cosign verify-blob "$BINARY" \
 <!-- 
 Cosign 2.0 requires the `--certificate-identity` and `--certificate-oidc-issuer` options.
 
-To learn more about keyless signing, please refer to [Keyless Signatures](https://docs.sigstore.dev/cosign/keyless).
+To learn more about keyless signing, please refer to [Keyless Signatures](https://docs.sigstore.dev/signing/overview/).
```
Comment (dipesh-rawat) [inline content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md:80]: It appears that updating the `Keyless Signatures` link on [Line 89](https://github.com/kubernetes/website/blob/3ca006943fb895d0876bbd5cd4017f4f66c8ff87/content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md?plain=1#L89) might also be needed.

Comment (windsonsea) [reply content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md:80]: Yes, that is required.

Comment (Arhell) [reply content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md:80]: Done

**Review by windsonsea — COMMENTED** [review]
/approve

**Review by my-git9 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**